src/number_theory/bernoulli.lean

```diff
@@ -204,3 +204,39 @@ begin
   { apply factorial_mul_factorial_dvd_factorial_add, },
   { exact cast_ne_zero.mpr hj', },
 end
+
+open ring_hom
+
+/-- Odd Bernoulli numbers (greater than 1) are zero. -/
+theorem bernoulli_odd_eq_zero : ∀ n : ℕ, (n % 2 = 1 ∧ 1 < n) → bernoulli n = 0 :=
```

Comment:
This is probably easier to work with as:
```suggestion
theorem bernoulli_odd_eq_zero (n : ℕ) (h_odd : n % 2 = 1) (hlt : 1 < n) : bernoulli n = 0 :=
```
Then you can remove the `rintros` and `cases` below